docs/cloud/faq/README.md

```diff
@@ -0,0 +1,34 @@
+# FAQ
+
+## No DNS Resolution of the cluster
+
+Some ISP, router and dns servers will not resolve or filter out answers to DNS queries for  `xxxx.mesdb.eventstore.cloud` because the address resolves to a private IP range.
+
+To check if this is the issue you have, perform the following `nslookup` replacing the domain by your cluster DNS name.
+``` bash
+nslookup buh63kqrh41nfqpviing.mesdb.eventstore.cloud
```

Comment:
**CONCERN:** Is `buh63kqrh41nfqpviing` random (not the real cluster DNS name)?

Reply:
> Some ISP, router and dns servers 
Some Internet providers, routers and DNS servers

Reply:
@oskardudycz  yes buh63kqrh41nfqpviing  is _NOT_ random , but it's in the public DNS system .... so it's discoverable anyway

Reply:
I think that it'd be better to randomise it a bit, just in case.